### DISCUSSION TITLE
Select media from the list: select all, select from-to

### DISCUSSION THREAD

**Discussion by Chris78c** [music-assistant/support#2572] [Feature requests and ideas]
I searched the forum but couldn't find anything, I hope the request hasn't already been made.
One very useful thing would be to be able to select Media faster and not one by one as now (both from App-Mobile and Browser).
In the video I included below I took the File Manager App as an example. 
You all already know these functions.
Select all, select from - to.

I find that having these functions in Music assistant would make it much easier to create playlists.

Thank you very much for your attention.  

https://github.com/music-assistant/hass-music-assistant/assets/165342443/99039ec3-c4d5-4402-b384-c3b3fae1d048

**Comment by jpbaril** [comment]
Agree, I was trying to select all items in a long list so to then manually deselect just a few to make quicker selection, but I right now I have to manually select one by one.

**Reply by OzGav** [reply]
You can use CTRL-A

**Reply by Chris78c** [reply]
Ok thank you but from smartphone is it impossible (Ctrl-a) and then select "from-to" would a good Feature too...

**Reply by OzGav** [reply]
From 2.5.0 beta and soon the 2.4 stable there is a new icon which will appear when the multi select icon is pressed which allows SELECT ALL
